lightning/src/ln/offers_tests.rs

```diff
@@ -508,15 +508,15 @@ fn creates_and_pays_for_offer_using_two_hop_blinded_path() {
 	features.set_onion_messages_optional();
 	features.set_route_blinding_optional();
 
-	let chanmon_cfgs = create_chanmon_cfgs(6);
-	let node_cfgs = create_node_cfgs(6, &chanmon_cfgs);
+	let chanmon_cfgs = create_chanmon_cfgs(7);
+	let node_cfgs = create_node_cfgs(7, &chanmon_cfgs);
 
 	*node_cfgs[1].override_init_features.borrow_mut() = Some(features);
 
 	let node_chanmgrs = create_node_chanmgrs(
-		6, &node_cfgs, &[None, Some(accept_forward_cfg), None, None, None, None]
+		7, &node_cfgs, &[None, Some(accept_forward_cfg), None, None, None, None, None]
 	);
-	let nodes = create_network(6, &node_cfgs, &node_chanmgrs);
+	let nodes = create_network(7, &node_cfgs, &node_chanmgrs);
```

Comment:
Could you change this back to use six nodes?

Reply:
Looks like I missed removing it when I removed the extra channel.
Thanks for catching it! 🌟
Fixed in [pr3163.08](https://github.com/shaavan/rust-lightning/commits/pr3163.08)